content/en-us/projects/assets/api.md

```diff
@@ -1,195 +1,318 @@
 ---
-title: External Catalog Queries
-description: External Catalog Queries let you externally query the Creator Store Catalog and Marketplace Catalog.
+title: Creator Store Queries
+description: Creator Store Queries let you externally query the Creator Store and Marketplace Catalog.
 ---
 
-You can search Roblox's assets outside Studio by accessing the external catalog API. Use the [Creator Store API](#creator-store-api) to query Studio assets, such as meshes, models, and audio, and the [Marketplace API](#marketplace-api) to query avatar assets on the Marketplace.
+You can search Roblox's assets outside Studio by accessing the Creator Store API. Use the [Creator Store API](#creator-store-api) to query Studio assets, such as meshes, models, and audio, and the [Marketplace API](#marketplace-api) to query avatar assets on the Marketplace.
 
 Each API requires a URL and custom search parameters for that specific catalog. If both URL and parameters are valid, the API returns a JSON format with the results of your search.
 
 ## Creator Store API
 
+> [!NOTE]
+> We acknowledge that the toolbox-service API described below is not covered on the official Cloud APIs documentation. We are planning to better support Creator Store search with a cleaner API in future, stay tuned for updates.
```

Comment:
Is there a specific reason why it's not being added to the official Cloud APIs doc? If so, it might be good to provide more specifics here instead of a promise for future updates.

Reply:
It requires more planning to add to the official api docs. An internal team at roblox is working on getting a better estimate for this.